src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts

```diff
@@ -4,6 +4,7 @@ import { GetScheduledEventUsers } from "../../../types/guilds/scheduledEvents.ts
 import { GuildMember } from "../../../types/members/guildMember.ts";
 import { User } from "../../../types/users/user.ts";
 import { Collection } from "../../../util/collection.ts";
+import { validateLength } from "../../../util/mod.ts";
```

Comment:
Why this import when its not used?

Reply:
oops I was going to use it then realize limit is a number and not a string so yeah let me remove it real quick